src/Rules/Complexity/ForbiddenInlineClassMethodRule.php

```diff
@@ -65,6 +65,11 @@ public function processNode(Node $node, Scope $scope): array
             }
 
             $onlyStmt = $classMethod->stmts[0] ?? null;
+
+            if ($onlyStmt !== null && $onlyStmt->getAttribute('startLine') !== $onlyStmt->getAttribute('endLine')) {
+                continue;
+            }
```

Comment:
This is not a type check, but only line position check.

Reply:
Yes, because that was my goal. But after thinking more about it, i think that first example in issue (https://github.com/symplify/phpstan-rules/issues/37) is not proper change, and in that case it also should throw an error. So now I think, that only second example (with builder) should not throwing error. What do You think about it?

Reply:
Yes, because that was my goal. But after thinking more about it, i think that first example in issue (https://github.com/symplify/phpstan-rules/issues/37) is not proper change, and in that case it also should throw an error. So now I think, that only second example (with builder) should not throwing error. What do You think about it?